dig/doc.go

```diff
@@ -20,7 +20,7 @@
 
 // Package dig is the Dependency Injection Graph.
 //
-// Package dig provides a fairly opinionated way of resolving object dependencies.
+// package dig provides an opinionated way of resolving object dependencies.
```

Comment:
Rant alert :P I think we overuse the word opinionated, I don't see dig as opinionated particularly. Each type of DI provides a different way of resolving dependencies that's all.

Why not "Dig resolves dependencies and provides users with an instantiated object when requested"